data/strings/types_strings.txt

```diff
@@ -29855,6 +29855,41 @@
   [type.tourism.attraction.specified]
     ref = type.tourism.attraction
 
+  [type.tourism.camp_pitch]
+    en = Camp Pitch
```

Comment:
It looks like `en` refers to American English? (I'm not an Organic Maps dev) This checks out below: "Campground" in `en-US` translates to "Campsite" in `en-GB`.

Well, with that in mind, what British folks refer to as a "Camp Pitch", us Americans refer to, confusingly enough, as a "Campsite". ("Camp Pitch" is not a phrase I'd heard of before looking this up the other day, and it made it hard to find this feature in the OSM taxonomy). This is currently mentioned in the opening paragraph here: https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcamp_site

Reply:
Yes, "camp pitch" is a term I have never, ever heard in the US.

Reply:
I also hadn't heard that phrase but was trying to go with the flow of how OSM classifies things. I'm an American and 100% would call it a camp site, but I know that that was getting repetitive and I couldn't come up with something unambiguous for the two. I'll keep things Americanish for the Americans, I guess! I'll adjust with the next force push.